django-entrypoint.sh

```diff
@@ -31,6 +31,11 @@ if [ "$1" = 'gunicorn' ]; then
     su-exec django django-admin migrate --noinput
   fi
 
+  # Allow running of collectstatic command because it might require env vars
+  if [ -n "$RUN_COLLECTSTATIC" ] && [$RUN_COLLECTSTATIC == 'True']; then
```

Comment:
Everywhere else, we expect the envvar to either be set (for true) or unset (for false). Do we want to be different here?

Reply:
@jerith I'm concerned that someone who wants to turn this off might just change the env var to "false" and then it might not be clear to them that it's still running

Reply:
Though this has the opposite problem, where someone might set it to "true" or "yes" and it might not be clear that it isn't running. I'm open to suggestions.

Reply:
That's already the case for everything else, and checking specifically for `'True'` means `true` and `1` and `yes` won't work...

Reply:
I think just do what the other checks in the script are doing. There isn't really a common standard for this in bash and whenever I'm trying to control things with envvars I end up reading the code to see what it expects anyway. 🤷

Reply:
True, but none of the other fields are True or False values. There's another piece of information being provided and we make the decision based on it's presence or absence.

Reply:
`$SKIP_MIGRATIONS`?

Reply:
True. literally the one I was basing this work on as well :see_no_evil: